<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pkgselect</class>
 <widget class="QWidget" name="pkgselect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Package Selection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton {
    spacing: 13px;
}

QCheckBox {
    spacing: 13px;
}

QLabel {
    padding-left: 33px;
    padding-right: 33px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1">
    <widget class="QLabel" name="additional_label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 0px; padding-right: 0px;</string>
     </property>
     <property name="text">
      <string>Additional Options</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="left_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="normal_text">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Web browser, utilities, office software, games, and media players</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="updates_button">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download and install updates following installation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QRadioButton" name="minimal_button">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minimal Installation</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="normal_button">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Normal Installation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <spacer name="stretch_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="apps_label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 0px; padding-right: 0px;</string>
     </property>
     <property name="text">
      <string>Installation Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="right_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="full_button">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Full Installation</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QScrollArea" name="extraparty_scroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="extraparty_scrollhouse">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>758</width>
        <height>174</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="packages_layout"/>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="full_text">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>All applications in the Normal Installation, and all extra third-party packages listed below</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <spacer name="pushup">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="minimal_text">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Only the desktop environment</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0" colspan="3">
    <widget class="QLabel" name="mandatory_warning_label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Note: MamoLinux and flavors are NOT responsible for third-party software installed from this list.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="extraparty_text">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 0px; padding-right: 0px;</string>
     </property>
     <property name="text">
      <string>Install additional third-party packages</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="updates_text">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>This saves time after installation, and keeps your system secure</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
